### PR TITLE
updating LocationStack explanation

### PR DESCRIPTION
**Pull request by vincenthoogsteder**
better explaining that LS is a hierarchy of UI as the user sees it

### CODE REVIEW

File: docs/docs/taxonomy/reference/location-contexts/overview.md [hunk 1]
```diff
@@ -7,9 +7,8 @@ title: Overview
 import Mermaid from '@theme/Mermaid';
 
 LocationContexts are used to populate a [Tracker's](/tracking/core-concepts/trackers) or 
-[Event's](/tracking/core-concepts/events.md) `location_stack` properties. A Location Stack is 
-meant to describe accurately where an [Event](/tracking/core-concepts/events.md) originated in the UI, e.g. 
-Sections, Menus, etc.
+[Event's](/tracking/core-concepts/events.md) `location_stack` properties.  A Location Stack is a visual representation of the UI location where a user is when an [Event](/tracking/core-concepts/events.md) occurs.  It is composed of a hierarchical stack of UI elements, meaning that the 
+order in the stack is the order in the UI as well.
```
Comment (ivarpruijn) [inline docs/docs/taxonomy/reference/location-contexts/overview.md:11]: Nice. Small suggestion (for both pages):
```suggestion
[Event's](/tracking/core-concepts/events.md) `location_stack` properties.  A Location Stack is a visual representation of the UI location for a user when an [Event](/tracking/core-concepts/events.md) occurs. It is composed of a hierarchical stack of UI elements, meaning that the order in the stack is the order in the UI as well.
````

Comment (vincenthoogsteder) [reply docs/docs/taxonomy/reference/location-contexts/overview.md]: updated

**Review by ivarpruijn — APPROVED** [review]

File: docs/docs/taxonomy/reference/location-contexts/overview.md (hunk 1 above)
Comment (jansenbob) [reply docs/docs/taxonomy/reference/location-contexts/overview.md]: I find this confusing. I think the word _visual_ is in the wrong place. A location stack is not a visual representation imho. That sounds like it is an image or diagram of some sort. 

A location stack represents from which exact visual UI location an event occurs.

**Comment by vincenthoogsteder** [comment]
I like that

**Comment by jansenbob** [comment]
Decided to tackle this once the taxonomy update is in place

**Pull request closed without merging**